### PR TITLE
chore: new Mayastor contributing.md

### PR DESCRIPTION
**Pull request by orville-wright**
chore: new Mayastor contributing.md
Signed-off-by: Orville Wright <orville.wright@yahoo.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,146 +1,143 @@
...
 
-- None, yet!
+---
+### Committing
 
-### Organization FAQs
+You must start you Development work off the `develop` branch. Do **Not use `master`.** - (We do not use that branch name.)
 
-- **What is a _Maintainer_?**
+| [<img src="https://github.com/openebs/community/blob/develop/images/rustacean-flat-gesture.png" width="200">](https://www.rust-lang.org/)  | :star: We run the [bors][bors] ```PR Merge bot tool```.<BR>:star: Bors will merge your commits once they has passed review.<BR>:star: Please ask if you need to [_squash merges_][squash-merges]. We generally do this. It's best to ask first.<BR>:star: Each commit message must adhere to [Conventional Commits][conventional-commits].<BR>:star: You can use [`convco`][tools-convco] if you need a tool to help with commit msgs.<BR>:star: It O.K to force push your branch if you need. Feel free to rewrite commit history it needed. |
```
Comment (dsharma-dc) [inline CONTRIBUTING.md:78]: I don't much on markdown syntax but I guess these content in square braces must point to some link, e.g. `bors` , `Conventional Commits` etc

**Review by dsharma-dc — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (orville-wright) [reply CONTRIBUTING.md:78]: Yup. So those links were from the original MayData created doc. 
I've tested them all,  in the original doc and they do point to real websites.... but for some reason the links don't work on the branch when I'm hacking on it. (not sure why).
For example...  [Conventional Commits][conventional-commits] links out to -> https://www.conventionalcommits.org/en/v1.0.0/
- I'm not sure how GitHub converts "[conventional-commits]" to "https://www.conventionalcommits.org/en/v1.0.0/
" but it does. Which is weird, but it works.

**Comment by orville-wright** [comment]
Hey @avishnu, can you give my PR a quick review and approve. I'd like to make this page change live to support our CNCF re-apply application.

**Review by avishnu — APPROVED** [review]

**Comment by orville-wright** [comment]
bors merge

**Pull request merged**